ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config

```diff
@@ -0,0 +1,29 @@
+scripts=[
```

Comment:
This should be moved back to regular ACLs set via the content package.

Reply:
Do you mean this entire repo unit? This script is to prevent ERRORs on bundle/component start. Ultimately they will be in repoinits anyhow..?

Reply:
double-checked that its fine (preferred) to use repoinit to manage the mutable space, so unless there are other concerns i vote to keep it as repoinit (since that's the ideal state anyhow).

Reply:
oh yes - sorry, did a deployment to make sure it was ok. Let me check on it.
 
I don't think we can even put /etc/tags into ui.content package, since the mere presence of that node causes TagManager API to use it (/etc/tags) as the tag root.